modules/op-configuring-pipelines-control-plane.adoc

```diff
@@ -0,0 +1,51 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: PROCEDURE
+[id="op-configuring-pipelines-control-plane_{context}"]
+= Configuring the {pipelines-title} control plane
+
+[role="_abstract"]
+You can customize the {pipelines-shortname} control plane by setting the configuration fields in the `TektonConfig` custom resource (CR). {pipelines-title} Operator automatically adds the fields with default values, if you do not set them in the `TektonConfig` CR.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Administration* → *CustomResourceDefinitions*.
+
+. Use the *Search by name* box to search for `tektonconfigs.operator.tekton.dev` custom resource definition (CRD). Click *TektonConfig* to see the CRD details page.
+
+. Click the *Instances* tab and select *config* to see `TektonConfig` CR details.
```

Comment:
@shipsing After the user clicks the *Instances* tab, from that page they choose *config*, not from the dropdown list. Is my wording here incorrect? What would be the right wording then?
Here is how it looks in the console:
![image](https://github.com/openshift/openshift-docs/assets/98941675/d5ed7c37-dadd-4434-82f6-d630177c4c71)

Reply:
Since we do not have any list to select from. We are actually clicking the instance name `config`, instead of selecting.
. Click the *Instances* tab.
. Click the *config* instance to see the`TektonConfig` CR details.

Reply:
Thanks for the clarification!